contrib/styles.scss

```diff
@@ -0,0 +1,298 @@
+
+a,
+a:visited {
+  color: inherit;
+}
+
+.code {
+  @include outer-container();
+  background-color: $lightblue;
+  padding:0.5em;
+  text-align: center;
+}
+
+.page-section {
+  @include outer-container;
+  background-color: $lightblue;
+  line-height: $base-line-height * 1.5;
+  margin-bottom: 3em;
+  text-align: center;
+}
+
+.page-section__inner {
+  background-color: $blue;
+  font-size: 0.8em;
+  height: $base-line-height * 4rem;
+  line-height: $base-line-height;
+  margin-bottom: 2em;
+  padding: 1em;
+  text-align: left;
+}
+
+// Zero
+// ============================================
+
+.zero__alpha,
+.zero__beta {
+  @include span-columns(6);
+  height: $base-line-height * 4rem;
+  line-height: $base-line-height;
+  padding: 1em;
+  text-align: left;
+}
+
+// First
+// ============================================
+
+.first__alpha {
+  @include span-columns(1);
+}
+
+.first__beta {
+  @include span-columns(11);
+}
+
+.first__gamma {
+  @include span-columns(2);
+}
+
+.first__delta {
+  @include span-columns(10);
+}
+
+.first__epsilon {
+  @include span-columns(3);
+}
+
+.first__zeta {
+  @include span-columns(9);
+}
+
+.first__eta {
+  @include span-columns(4);
+}
+
+.first__theta {
+  @include span-columns(8);
+}
+
+.first__iota {
+  @include span-columns(5);
+}
+
+.first__kappa {
+  @include span-columns(7);
+}
+
+.first__lambda {
+  @include span-columns(6);
+}
+
+.first__mu {
+  @include span-columns(6);
+}
+
+.first__beta,
+.first__delta,
+.first__zeta,
+.first__theta,
+.first__kappa,
+.first__mu {
+  @include omega;
+}
+
+// Second
+// ============================================
+
+.second__alpha {
+  @include span-columns(4);
+}
+
+.second__beta {
+  @include span-columns(8);
+  line-height: $base-line-height * 2;
+  padding: 0;
+  text-align: center;
+}
+
+.second__beta__alpha,
+.second__beta__beta {
+  @include span-columns(4 of 8);
+  background-color: $lightblue;
+  height: 84%;
+  padding: 1em;
+}
+
+// Third
+// ===================================
+
+.third__dummy {
+  border-bottom: 0;
+  height: 5.5em;
+  text-align: center;
+}
+
+.third__alpha {
+  @include fill-parent();
+  @include row(table);
+  margin-bottom: 0;
+  padding: 0;
+}
+
+.third__alpha__alpha {
+  @include span-columns(4);
+  border-right: 1px solid rgba(255,255,255, 0.5);
+}
+
+.third__alpha__beta {
+  @include reset-display;
+  @include span-columns(8);
+  height: 20em;
+}
+
+// Fourth
+// ===================================
+
+.fourth {
+
+  .fourth__alpha {
+    @include span-columns(6);
+    @include shift(3);
+    height: 20em;
+    text-align: center;
+  }
+}
+
+// Fifth
+// ===================================
+
+.fifth__box {
+  @include omega(4n);
+  @include span-columns(3);
+  font-size: 0.78em;
+  height: 8em;
+  margin-top: 2em;
+}
+
+// Fifth
+// ===================================
+
+.sixth {
+
+  pre {
+    padding: 10px 0;
+  }
+
+  @include media($medium-screen) {
+    font-size: 0.9em;
+    line-height: 1.5em;
+  }
+}
+
+.sixth__alpha,
```

Comment:
Selector `sixth__alpha` should be written in lowercase with hyphens<br>Selector `sixth__beta` should be written in lowercase with hyphens